examples/timeSeries-hand-gestures/sketch.js

```diff
@@ -0,0 +1,192 @@
+/*
+ * 👋 Hello! This is an ml5.js example made and shared with ❤️.
+ * Learn more about the ml5.js project: https://ml5js.org/
+ * ml5.js license and Code of Conduct: https://github.com/ml5js/ml5-next-gen/blob/main/LICENSE.md
+ *
+ * This example demonstrates training a Hand Gesture classifier through ml5.TimeSeries.
+ */
+
+let seqLength = 50;
+
+let handPose;
+let video;
+
+let hands = [];
+let sequence = [];
+
+let recordingFinished = false;
+let predictedWord = "";
+
+// UI variables
+let trainingWords = {};
+
+function preload() {
+  // Load the handPose model
+  handPose = ml5.handPose();
+
+  // setup the timeseries neural network
+  let options = {
+    outputs: ["label"],
+    task: "classification",
+    dataMode: "spatial",
+    debug: "true",
+    learningRate: 0.001,
+  };
+  model = ml5.timeSeries(options);
+}
+
+function setup() {
+  createCanvas(640, 480);
+
+  // setup video capture
+  video = createCapture(VIDEO);
+  video.size(640, 480);
+  video.hide();
+
+  // place UI elements
+  UI();
+
+  // use handpose model on video
+  handPose.detectStart(video, gotHands);
+}
+
+function draw() {
+  // draw video on frame
+  image(video, 0, 0, width, height);
+
+  drawPredictedWord();
+
+  // if hands are found then start recording
+  if (hands.length > 0 && recordingFinished == false) {
+    if (sequence.length <= seqLength) {
+      // get coordinates from hands (21 points)
+      handpoints = drawPoints();
+      sequence.push(handpoints);
+
+      // once sequence reaches the seqLength, add sequence as just one X value
+    } else if (sequence.length > 0) {
+      // get the training word from the input box
+      let train_word = nameField.value();
```

Comment:
Stylistic nitpick: perhaps `trainWord`, since the rest of the example uses camelCase